examples/vue/src/pages/ui/components/authenticator/sign-up-with-email-lambda/index.vue

```diff
@@ -10,7 +10,7 @@ Amplify.configure(aws_exports);
 <template>
   <authenticator initial-state="signUp">
     <template v-slot="{ user, signOut }">
-      <h1>Hello {{ user.username }}!</h1>
+      <h1>Hello {{ user.attributes?.email }}!</h1>
```

Comment:
nit: if `user.attributes` is undefined, there might not need a space between "Hello" and "1"...


```suggestion
      <h1>Hello{{ user.attributes? : `${ user.attributes.email}!`: "!"}}!</h1>
```

Reply:
Interesting, I'm going to leave it as is on this one.

Reply:
This is a fair comment, but I think it's more of a type discrepancy in Authenticator. `user.attributes.email` will always be defined for email use cases, but TS doesn't know that. So I'm good with leaving this as is as @ErikCH has mentioned.